src/web/services/log.service.ts

```diff
@@ -17,6 +17,7 @@ import { HttpRequestService } from './http-request.service';
   providedIn: 'root',
 })
 export class LogService {
+  static readonly LOG_DATE_TIME_FORMAT: string = 'ddd, DD MMM YYYY hh:mm:ss A';
```

Comment:
This string is also used in `instructor-track-view-page.component.ts`. Perhaps it would be good to update it to use this constant as well for consistency.

Reply:
Oops my bad, I missed out those strings due to the additional comma in them. The changes have been made.